Update replace-brick documentation (#782)
gluster/glusterdocs: docs/Administrator-Guide/Managing-Volumes.md

           Status of volume: r2
           Gluster process                        Port    Online    Pid
           ------------------------------------------------------------------------------
           Brick Server1:/home/gfs/r2_5            49156    Y    5731 <---- new brick is online
           Brick Server2:/home/gfs/r2_1            49153    Y    5354
           Brick Server1:/home/gfs/r2_2            49154    Y    5365
           Brick Server2:/home/gfs/r2_3            49155    Y    5376

    - Users can track the progress of self-heal using: `gluster volume heal [volname] info`, or by checking the size of the new brick.

>     - `# gluster volume heal <volname> info` will show that no heal is required when the data on the replaced brick has been migrated completely.

Reviewer: ```suggestion
    - `# gluster volume heal <volname> info` will show that no heal is required when the data is fully synced to the replaced brick.
```

The data will be synced from the other good brick(s) of the replica to the replaced one. The current statement gives the impression that the data will be migrated from the old brick to the newly replaced brick.
